ChangeLog

```diff
@@ -1,3 +1,10 @@
+23.3.3
+ - Fix pip-managed ansible on pip < 23.0.1 (#4403)
+
+23.3.2
+ - Revert "ds-identify/CloudStack: $DS_MAYBE if vm running on vmware/xen (#4281)"
+   (#4511) (LP: #2039453)
+
```

Comment:
@holmanb not strictly a blocker to this PR, but since we've sync'd our upstream Changelog changes representing 23.3.3 and 23.3.2, it may be a good time to also sync the "released" debian/changelog entries from upstream/ubuntu/mantic-23.3.x as well into debian/changelog in this branch too same goes for ubuntu/jammy|focal|lunar branches. Or we can do that sync prior to our next official publication as we are leaving this d/changelog in UNRELEASED state